diff --git a/RobotRaconteurCore/CMakeLists.txt b/RobotRaconteurCore/CMakeLists.txt
index 9254e2c..ebf402a 100644
--- a/RobotRaconteurCore/CMakeLists.txt
+++ b/RobotRaconteurCore/CMakeLists.txt
@@ -218,17 +218,17 @@ if(BUILD_SHARED_LIBS)
     set(RobotRaconteur_SHARED_CORE_VERSION "${RobotRaconteur_LIB_VERSION}")
 endif()
 
-if(BUILD_SHARED_LIBS)
-    set(CMAKE_DEBUG_POSTFIX "d")
-    add_library(RobotRaconteurCore SHARED ${RobotRaconteurCore_src} ${RobotRaconteurCore_header})
-    set_target_properties(
-        RobotRaconteurCore
-        PROPERTIES VERSION "${RobotRaconteur_SHARED_CORE_VERSION}" SOVERSION "${RobotRaconteur_SHARED_CORE_VERSION}"
-                   COMPILE_DEFINITIONS "ROBOTRACONTEUR_CORE_EXPORTS")
-else()
+# if(BUILD_SHARED_LIBS)
+#     set(CMAKE_DEBUG_POSTFIX "d")
+#     add_library(RobotRaconteurCore SHARED ${RobotRaconteurCore_src} ${RobotRaconteurCore_header})
+#     set_target_properties(
+#         RobotRaconteurCore
+#         PROPERTIES VERSION "${RobotRaconteur_SHARED_CORE_VERSION}" SOVERSION "${RobotRaconteur_SHARED_CORE_VERSION}"
+#                    COMPILE_DEFINITIONS "ROBOTRACONTEUR_CORE_EXPORTS")
+# else()
     add_library(RobotRaconteurCore STATIC ${RobotRaconteurCore_src} ${RobotRaconteurCore_header})
     set_target_properties(RobotRaconteurCore PROPERTIES PREFIX lib)
-endif()
+# endif()
 
 target_link_libraries(
     RobotRaconteurCore
@@ -281,10 +281,10 @@ if(WIN32)
         target_compile_options(RobotRaconteurCore PRIVATE "-mthreads")
     endif()
 
-    if(BUILD_SHARED_LIBS)
-        target_compile_definitions(RobotRaconteurCore PRIVATE ROBOTRACONTEUR_CORE_EXPORTS
-                                   INTERFACE ROBOTRACONTEUR_CORE_IMPORTS)
-    endif()
+    # if(BUILD_SHARED_LIBS)
+    #     target_compile_definitions(RobotRaconteurCore PRIVATE ROBOTRACONTEUR_CORE_EXPORTS
+    #                                INTERFACE ROBOTRACONTEUR_CORE_IMPORTS)
+    # endif()
 endif()
 
 if(APPLE)
